docs/source/deployment_guide_overview.md

```diff
@@ -0,0 +1,173 @@
+# Deploying a production network
+
+This deployment guide is a high level overview at the proper sequence for setting up a functioning Fabric network, in addition to best practices and a few of the many considerations to keep in mind when deploying. Note that this topic will discuss "setting up the network" as a holistic process from the perspective of a single individual. More likely than not, real world production networks will not be set up by a single individual but as a collaborative effort directed by several individuals (a collection of banks each setting up their own components, for example) instead.
+
+The process for deploying a Fabric network is difficult and presumes an understanding of Public Key Infrastructure and managing distributed systems. If you are a smart contract or application developer, you should not need this level of expertise in deploying a production level Fabric network. However, you might need to be aware how networks are deployed in ordering to develop effective smart contracts and applications.
+
+If all you need is a development environment to test chaincode, smart contracts, and applications against, we have a [Test Network](test_network.html) that can be deployed easily to a local environment. It includes two organizations, each owning one peer, and a single ordering service organization that owns a single ordering node. **This test network is not meant to provide a blueprint for deploying production components, and should not be used as such, as it makes assumptions and decisions that production deployments will not make.**
+
```

Comment:
I agree with the aim of the last line; however as a developer I would start to wondering what the differences are. 

How much fidelity is there between the test-network and production? i.e. how likely is something to work on test and then fail in production.

Reply:
I don't think this is a good place to talk about what the test network isn't. Better for the test network doc to mention the things about it that aren't production.